src/layout/403.tsx

```diff
@@ -0,0 +1,33 @@
+import { __ } from '@wordpress/i18n';
+
+const Forbidden = () => {
```

Comment:
Need to implement the **404** and **403** components designed by the @getdokan/product-plugin-team